src/models/ReceiptModel.java

```diff
@@ -0,0 +1,35 @@
+package models;
+
+public class ReceiptModel {
+
+    double sumOfPrices = 0;
+    double tax = 0;
+    double grandTotal;
+
+    public void updateTotal(Double individualItemPrice) {
```

Comment:
💡 I would assume you will add to this later, but just to tee it up.  I would anticipate that we would want to store the list of items in this receipt model as well.  This would likely impact how we know/update the grand total.

What you are doing here is keeping a running total, which works for now, but can introduce complications later. (think if you want to apply a coupon code that only affects pizzas purchased.  In your current implementation, you would not know how to modify the order/receipt after the item gets added. (i.e. you need to know that the customer has a coupon ahead of time in order to get the price added to the grand total correctly the first time.) Sometimes customers can suck and say oh yeah, hold up,  I have a coupon that i forgot to give you, which at that time your implementation would be F'd.  So instead what we can do is keep track of everything separately in the receipt model.  This would mean you can create a field on the model for items purchased, a field for coupon(s) to apply to the order, a field for current tax rates, a field for employee that is ringing up the order, a field for the customer id/name, etc.   THEN, when you want the grand total you invoke a function named calculateGrandTotal() which looks to all the information that should now be in the receipt model object and apply your grand total algorithm to calculate the grand total on the fly.  Note that in this new approach, you would not have a field for grand total, however there would be a function that knows how to calculate it every time.   When you store the data in a database long long down the road, we WOULD store the grand total in the DB, but we would NOT store it in the model object used in your JAVA program.

This was long winded, and as mentioned not needed at the moment, but a good exercise to start thinking about.